Modules/Core/Repositories/Eloquent/EloquentBaseRepository.php

```diff
@@ -191,4 +191,26 @@ public function clearCache()
     {
         return true;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function where($field, $value, $operator=null)
+    {
+        if (null==$operator) {
+            $operator='=';
+        } else {
+            list($value, $operator) = [$operator, $value];
+        }
+
+        return $this->model->where($field, $operator, $value);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function with($relationships)
```

Comment:
This is plural, so it's an array? If so, can you typehint the array please?

Reply:
`with` method accept string or array, if you typehint the array could be a problem

Reply:
I see ok, no need then 👍

Reply:
I used plural just for readability, no intent to let it be understood it is an array